services/business/workloads.go

```diff
@@ -22,11 +22,14 @@ func (in *WorkloadService) GetWorkloadList(namespace string) (models.WorkloadLis
 	for _, deployment := range deployments.Items {
 		selector, _ := in.k8s.GetDeploymentSelector(namespace, deployment.Name)
 		dPods, _ := in.k8s.GetPods(namespace, selector)
-		casted := &models.WorkloadOverview{}
+
+		casted := &models.WorkloadListItem{}
```

Comment:
"casted" is wrong in English, I guess, as to cast is irregular (cast/cast/cast).
I guess I saw this one in several places as well.